source/views/sis/course-search/search.js

```diff
@@ -100,10 +88,10 @@ function executeSearch(args: {
 
 	const results = filteredCourses.filter(
 		course =>
-			keywordSearch(query, course.name.toLowerCase()) ||
-			keywordSearch(query, (course.title || '').toLowerCase()) ||
+			keywordSearch(query, course.name.toLowerCase(), 1) ||
+			keywordSearch(query, (course.title || '').toLowerCase(), 1) ||
 			(course.instructors || []).some(name =>
-				keywordSearch(query, name.toLowerCase()),
+				keywordSearch(query, name.toLowerCase(), 1),
```

Comment:
Aren't these "1" parameters set to "2" in the current implementation? Does that matter?

Reply:
Yeah I changed the package to use <= instead of < so that that parameter is the permitted hamming distance.

Reply:
Ah I missed that change